vuepress/docs/next/tutorials/create/ms/run-local.md

```diff
@@ -2,159 +2,92 @@
+The form to enter the name and location for conferences should open in your browser. You may enter and save new data to see it in the other MFEs.
 
-4. When the widget starts, a browser window opens with the widget URL
-
-5. If you’re not logged in, you'll be redirected to the login page
-
-6. Log in using:
-
-        Username: user
-        Password: user
-
-7.  Once logged in, you can see the widget details with the ID 1 loaded
+>If you want to modify a different row in the database, edit the `microfrontends/conference-form/public/index.html` file. Change the `id` attribute in this line:
+``` html
+   <conference-form id="1" />
+```
 
-#### Widget Details notes:
+### Run the conference-details MFE
 
-If you want to modify a different row in the database, edit the `index.html` file in the public folder `ui/widgets/YOUR-ENTITY-NAME/detailsWidget/public`. Change the `id` attribute in this line:
+1. If you are running the previous micro frontend, click `Ctrl+C` to end the process
 
-    <my-entity-details service-url="%REACT_APP_SERVICE_URL%" id="1" />
+2. Start the conference-details MFE
+``` sh
+ent bundle run conference-details
+```
+When the run is complete, you should see the details MFE showing the information for the Conference with ID 1.
 
+>If you want to modify a different row in the database, edit `microfrontends/conference-details/public/index.html` file. Change the `id` attribute in this line:
+``` html
+    <conference-details id="1" />
+```
 
 ### Keycloak Settings and Issues 
-1. Change Keycloak Dev Settings
-
-If you want to use another Keycloak installation or change the service-URL of your widget, change the parameters set in the `.env.local` file generated by the Entando Blueprint in the root folder of your widgets. For instance, `ui/widgets/YOUR-ENTITY-NAME/tableWidget/.env.local`
-
-By default these variables are set to:
+1. Change Development Settings
 
-    REACT_APP_SERVICE_URL=http://localhost:8081/services/YOUR-APPLICATION-NAME/api
-    REACT_APP_KEYCLOAK_URL=http://localhost:9080/auth
-    REACT_APP_KEYCLOAK_REALM=jhipster
-    REACT_APP_KEYCLOAK_CLIENT_ID=web_app
+   If you want to use another Keycloak installation, modify the reference in the Keycloak YAML file located in the `svc/` folder.
 
-2. By default, Docker Compose recreates the Keycloak container and resets the H2 database each time it is started. To persist Keycloak changes across restarts, you can make the following updates: 
-   * In your src/main/docker/keycloak.yml file, replace\
-   `
-   '-Dkeycloak.migration.strategy=OVERWRITE_EXISTING',` with 
-   ```
-   '-Dkeycloak.migration.strategy=IGNORE_EXISTING',
-   ```   
-   * In the same file, add a persistent volume under `volumes`:
+2. In this blueprint generated project, Docker Compose persists Keycloak data across restarts by default. If you want your data to reset on restarts:
+   
+   * Edit the `svc/keycloak.yml` file. Replace `-Dkeycloak.migration.strategy=IGNORE_EXISTING`,
```

Comment:
why is the comma included in the edit? can just say replace ignore... with overwrite... ?

Reply:
comma is part of the code, it should be inside the backtick